bindings/src/querier.rs

```diff
@@ -461,7 +461,35 @@ impl<'a> ElysQuerier<'a> {
         let request: QueryRequest<ElysQuery> = QueryRequest::Custom(pools_query);
 
         let resp: QueryEarnPoolResponse = self.querier.query(&request)?;
-        Ok(resp)
+
+        match resp.pools {
+            Some(pools) => {
+                let pools_with_usd_values = pools
+                    .iter()
+                    .map(|pool| PoolResp {
+                        assets: pool
+                            .assets
+                            .iter()
+                            .map(|asset| {
+                                let price = self
+                                    .get_asset_price(asset.token.denom.clone())
+                                    .unwrap_or(Decimal::from_str("0").unwrap());
+                                PoolAsset {
+                                    token: asset.token.clone(),
+                                    weight: asset.weight,
+                                    usd_value: Some(asset.token.amount * price),
```

Comment:
Also, is this an actual USD value? Can you also verify that as well please? I have seen in other places that conversions go first through ASSET->USDC and then USDC->USD.

Reply:
https://github.com/elys-network/bindings/blob/78d9f2c5cf333ceceb916af9dba32acce0f9b8ee/bindings/src/querier.rs#L545-L547

Reply:
It should be correct, yes.

Reply:
price calc correct